dash_table/package-info.json

```diff
@@ -25,12 +25,15 @@
     "private::wait_js": "wait-on http://localhost:8080",
     "private::opentests": "cypress open",
     "private::runtests-v0": "cypress run --browser chrome --spec 'tests/cypress/tests/server/**/*'",
-    "private::runtests-v1": "cypress run --browser chrome",
+    "private::runtests-v1:unit": "cypress run --browser chrome --spec 'tests/cypress/tests/unit/**/*'",
+    "private::runtests-v1:standalone": "cypress run --browser chrome --spec 'tests/cypress/tests/standalone/**/*'",
+    "private::runtests-v1:server": "cypress run --browser chrome --spec 'tests/cypress/tests/server/**/*'",
+    "private::runtests-v1": "run-s private::runtests-v1:unit private::runtests-v1:standalone private::runtests-v1:server",
```

Comment:
Some annoying tidbits related to Cypress -- it seems the additional tests tipped it over into Heap Overflow land. Break up to test into sections for now. Should probably explore parallelisation in the future.